sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/lro/AnalyzeActions.java

```diff
@@ -40,12 +42,13 @@ public class AnalyzeActions {
      */
     public static void main(String[] args) {
         TextAnalyticsClient client = new TextAnalyticsClientBuilder()
-                                         .credential(new AzureKeyCredential("{key}"))
-                                         .endpoint("{endpoint}")
+                                         .serviceVersion(TextAnalyticsServiceVersion.V3_1)
+                                         .credential(new AzureKeyCredential(Configuration.getGlobalConfiguration().get("AZURE_TEXT_ANALYTICS_API_KEY")))
+                                         .endpoint(Configuration.getGlobalConfiguration().get("AZURE_TEXT_ANALYTICS_ENDPOINT"))
                                          .buildClient();
 
         List<TextDocumentInput> documents = new ArrayList<>();
-        for (int i = 0; i < 21; i++) {
+        for (int i = 0; i < 1; i++) {
```

Comment:
was this change intentional?

Reply:
Forget to revert the changes. Will revert the changes to samples